server/views/partials/layout.njk

```diff
@@ -14,89 +14,96 @@
-      n&&j.setAttribute('nonce',n.nonce||n.getAttribute('nonce'));
+            var n = d.querySelector('[nonce]');
+            n && j.setAttribute('nonce', n.nonce || n.getAttribute('nonce'));
 
-      f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','{{tagManagerContainerId}}');
-    </script>
-    <!-- End Google Tag Manager -->
-  {% endif %}
+            f.parentNode.insertBefore(j, f);
+          })(window, document, 'script', 'dataLayer', '{{ tagManagerContainerId }}');
+        </script>
+        <!-- End Google Tag Manager -->
+    {% endif %}
 
 
 
-  <!--[if !IE 8]><!-->
-  <link href="/assets/stylesheets/application.css?{{ version }}" rel="stylesheet"/>
-  <!--<![endif]-->
+    <!--[if !IE 8]><!-->
+    <link href="/assets/stylesheets/application.css?{{ version }}" rel="stylesheet" />
+    <!--<![endif]-->
 
-  <!--[if lt IE 9]>
-  <link href="/assets/stylesheets/application-ie8.css?{{ version }}" rel="stylesheet"/>
-  <script src="/assets/js/html5shiv-3.7.3.min.js"></script>
+    <!--[if lt IE 9]>
+  <link href='/assets/stylesheets/application-ie8.css?{{ version }}' rel='stylesheet'/>
+  <script src='/assets/js/html5shiv-3.7.3.min.js'></script>
   <![endif]-->
 
-  <script src="/assets/js/jquery.min.js"></script> 
-  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-          integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-          crossorigin="anonymous"></script>
-  <link href="https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet" crossorigin>
+    <script src="/assets/js/jquery.min.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
+            integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
+            crossorigin="anonymous"></script>
+    <link href="https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet" crossorigin>
 
 {% endblock %}
 
 <title>
-  {% block pageTitle %}  
-    {% if errors.length %}Error:{% endif %} {{pageTitle | default(applicationName)}} 
-  {% endblock %}
+    {% block pageTitle %}
+        {% if errors.length %}Error:{% endif %} {{ pageTitle | default(applicationName) }}
+    {% endblock %}
 </title>
 
 {% block header %}
-  {% include "./header.njk" %}
+    {% include "./header.njk" %}
 {% endblock %}
 
 {% block bodyStart %}
 {% endblock %}
 
     {% block beforeContent %}
-      {% if hasBackLink !== false %}
-        {{ govukBackLink({
-          text: "Back",
-          classes: "backLinkUrl",
-          attributes: {'data-qa': 'back-link'},
-          href: "#"
-        }) }}
-      {% endif %}
+        {% if hasBackLink !== false %}
+            {{ govukBackLink({
+                text: "Back",
+                classes: "backLinkUrl",
+                attributes: {'data-qa': 'back-link'},
+                href: "#"
+            }) }}
+        {% endif %}
 
-{% endblock %}
+        {% if breadCrumbs.enabled === true %}
```

Comment:
I did try just `{% if breadcrumbs %}` so we wouldn't need the flag. The flag seemed to work better though.

Suppose in the future, we could also infer the backlink is disabled if breadcrumbs are enabled. But thought I'd leave that for now 👀